lib/sass_spec/cli.rb

```diff
@@ -103,13 +103,6 @@ def self.parse
         options[:only_output_styles] << style
       end
 
-      opts.on("-r SPEC_DIR", "--root SPEC_DIR", "Root directory for the specs. ",
```

Comment:
Not sure spec directory is used by some of the libsass variants when skimming through https://github.com/search?p=1&q=org%3Asass+sass-spec&type=Code

Reply:
I see some stuff about configuring the location of the sass-spec gem itself, but nothing about using the --root flag.

Reply:
I don't have any insight into how our wrappers do their tests. We'd have to ask them cc @bolandrm @asottile @mgreter

Reply:
Also @scottdavis

Reply:
libsass-python doesn't invoke `sass-spec` currently (mostly leaning on `libsass` itself there) so this change is fine from our side!  Thanks for checking in :)

Reply:
perl-libsass has its own spec-runner at https://github.com/sass/perl-libsass/blob/develop/t/99_sass_specs.t

Reply:
@mgreter How hard would it be to move to the Ruby test runner (or, as libsass-python does, rely on LibSass's own test coverage)? I'm planning some refactors that will likely be challenging to port.

Reply:
Not possible since it would break the whole concept of cpan tests which mostly doesn't have python installed and also relies sometimes on GCC 4.4 : http://matrix.cpantesters.org/?dist=CSS-Sass

Reply:
Well, I guess consider this a heads-up that there will be some pretty substantial changes in test format coming soon.

Reply:
OK, not that I like to rewrite it for the third time ...

Reply:
That's why I was hoping it could re-use the Ruby runner :disappointed:. Another solution would be to just not run the language specs, since it's just a wrapper and LibSass already tests its language implementation thoroughly.

Reply:
I understand, but the cpan test matrix actually did uncover a few quirks here and there already which weren't catched by libsass CI before and it's always a good feeling when it passed the full matrix.